scripts/dotnet-cli-build/CompileTargets.cs

```diff
@@ -195,6 +195,10 @@ private static BuildTargetResult CompileStage(BuildTargetContext c, DotNetCli do
                 }
             }
 
+            // dotnet.exe is from stage0. But we must be using the newly built corehost in stage1
```

Comment:
Well stage1 dotnet.exe is not the locally built corehost. Instead it is copied from stage0 (due to `dotnet publish`). So any fix in corehost src code cannot be bootstrapped in our builds easily. So I am planning to leave it as is.